code/modules/ninja/ninja_event.dm

```diff
@@ -78,9 +78,7 @@ Contents:
 
 /proc/create_space_ninja(spawn_loc)
 	var/mob/living/carbon/human/new_ninja = new(spawn_loc)
-	new_ninja.randomize_human_appearance(~(RANDOMIZE_NAME|RANDOMIZE_SPECIES))
```

Comment:
Why remove the randomization?

Reply:
Because human is already randomised upon spawn